dsgrid/config/annual_time_dimension_config.py

```diff
@@ -70,73 +71,70 @@ def convert_dataframe(
         self,
         df,
         project_time_dim,
-        model_years=None,
-        value_columns=None,
+        value_columns: set[str],
+        scratch_dir_context: ScratchDirContext,
         wrap_time_allowed=False,
         data_adjustment=None,
     ):
-        assert model_years is not None
         assert value_columns is not None
         match self.model.measurement_type:
             case MeasurementType.MEASURED:
-                df = self.map_annual_time_measured_to_datetime(df, project_time_dim, model_years)
+                df = self.map_annual_time_measured_to_datetime(df, project_time_dim)
```

Comment:
I'm not sure that it makes sense for annual time to have a measurement type other than total. To have some other type, wouldn't we need to have frequency in the config?